automated install/basic-install.sh

```diff
@@ -701,8 +701,11 @@ update_pacakge_cache() {
 
   echo ":::"
   echo -n "::: Updating local cache of available packages..."
-  ${UPDATE_PKG_CACHE} &> /dev/null
-  echo " done!"
+  if eval ${UPDATE_PKG_CACHE} &> /dev/null; then
```

Comment:
Approved (tested on DO, installed and repaired fine)

Although I don't like this `eval` statement here, but I see why it is necessary.